layouts/html.hbs

```diff
@@ -145,6 +145,10 @@
             HitchhikerJS.RuntimeConfigReceiver.handle(message.runtimeConfig);
           } else if (window.isOverlay) {
             window.Overlay.onMessage(message);
+          } else if (message.eventType && message.eventType === 'click') {
+            ANSWERS.components._activeComponents
```

Comment:
We should probably make a method under `ANSWERS.components` called `getByName`. That way, we don't leak private, implementation details here. But, that's beyond this scope of this PR. I'll make a separate item to address.